### DISCUSSION TITLE
How to get the Nova home route?

### DISCUSSION THREAD

**Discussion by Synchro** [laravel/nova-issues#4562] [Q&A] [answered]
Nova defines `nova.index`, and much of the time that does point at Nova's home page (so it's an obvious thing to try), but its dynamic nature means that it doesn't always work, as in #3351. Responses there say that it's not meant for that and that we shouldn't use it. That's fine - but what should we use instead? What's a reliable way to get the Nova home route?

My best guess at the moment is `url(config('nova.path'))`. Is that a good choice?

**Comment by crynobone** [comment, marked as answer]
From outside Nova you can use `route('nova.pages.home')`. Otherwise you can use `Laravel\Nova\Nova::url('/')` or `Nova.url('/')` from JavaScript.
